packages/apps/tasks/README.md

```diff
@@ -0,0 +1,9 @@
+# Tasks
+
+## Development
+
+To run the app with HALO.
+
+```bash
+pnpm -w nx run tasks:serve-with-vault
```

Comment:
@wittjosiah how do I make this actually work?

Reply:
You need the `serve-with-vault` target in your `project.json` file, should be able to copy from other apps